### DISCUSSION TITLE
Change the colors on tool tips for node styles and mod times?

### DISCUSSION THREAD

**Discussion by JJSlote** [freeplane/freeplane#1481] [Help]
Hi - I've searched widely, but can't locate some of the styling for the tool tips that appear as we hover on a node. Node Styles and Modification Times are being displayed black on black under the Look and Feel themes _Flat Dark_, _Flat Darcula_, and _Atom One Dark Contrast._ (Attributes are fine; their styling is governed by the Attributes node under _Edit Styles_.)

Node styles and mod times not readable on hover:
![ToolTipDark](https://github.com/freeplane/freeplane/assets/68667676/bc38d2a7-04e9-474b-b80b-0bdbb914129d)
Readable when selected via Ctrl-A:
![ToolTipDarkSelected](https://github.com/freeplane/freeplane/assets/68667676/48c0d701-70b5-4ad1-81a7-0482533d47a7)

Anyone know where this set of stylings is configured?
Thanks -- Jerome

**Comment by dpolivaev** [comment]
It's a bug, I am fixing it.
